upnp/src/genlib/net/http/webserver.c

```diff
@@ -521,7 +521,7 @@ static int get_file_info(
 	/*! [out] File information object having file attributes such as filelength,
 	 * when was the file last modified, whether a file or a directory and
 	 * whether the file or directory is readable. */
-	OUT UpnpFileInfo *info)
+	/* OUT */ UpnpFileInfo *info)
```

Comment:
The idea was to have those commented in the cases where the doxygen or documentation comment is missing them or there is no documentation comment, so that its easier to know later when writing the doxygen comments.
I must have accidentally left it there in some cases where the doxygen had it too…

Reply:
I've checked again and removed some more where they were already present in the doxygen/doc-comment thing.